client/state/inline-help/selectors.js

```diff
@@ -73,3 +73,13 @@ export function getInlineHelpCurrentlySelectedLink( state ) {
 	const result = get( results, getSelectedResultIndex( state ), null );
 	return get( result, 'link', '' );
 }
+
+/**
+ * Returns a bool indicating if the contact form UI is showing the Q&A suggestions.
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Boolean}        Is the contact form UI showing the questions
+ */
+export function isShowingQandAInlineHelpContactForm( state ) {
+	return get( state, 'inlineHelpSearchResults.contactForm.isShowingQandASuggestions' );
```

Comment:
Without a default value get will return `undefined` when `isShowingQandASuggestions` is missing and thus violate the `@return {Boolean}` doc statement.
We can use `get`s third argument to supply a default value of `true` or `false` to keep inline with the jsdocs here though.